### PR TITLE
push to wagicproject

### PR DESCRIPTION
**Pull request by zethfoxster**

### CODE REVIEW

File: projects/mtg/include/MTGRules.h [hunk 1]
```diff
@@ -58,6 +58,10 @@ class MTGEventBonus: public PermanentAbility
     int Angel[2];
     bool dragonbonusgranted[2];
     int dragon[2];
+	bool eldrazibonusgranted[2];
+	int eldrazi[2];
+	bool werewolfbonusgranted[2];
+	int werewolf[2];
```
Comment (xawotihs) [inline projects/mtg/include/MTGRules.h]: Your tabulations seem badly configured...

Comment (zethfoxster) [inline projects/mtg/include/MTGRules.h]: yeah, i know, i dont have my original set up which had the tabs set up correctly, and I dont remember what the correct settings are for wagic project. the thread on the forums was not very specific as to where you set this up.

Comment (kevlahnota) [inline projects/mtg/include/MTGRules.h]: hope this screenshot will help, just click it to zoom
screenshot 50

**Comment by zethfoxster** [comment]
quick question, any fixes that i do AFTER i open a pull request, are they included in this pull also, or do i need to do another pull after this one so fixes go in?

File: projects/mtg/include/MTGRules.h (hunk 1 above)
Comment (zethfoxster) [inline projects/mtg/include/MTGRules.h]: ok set them to those settings, :) corssing fingers and hoping it doesnt turn out wrong. what I dont get is why does it look fantastic in the compiler, but then in commit looks like garbage?

**Comment by kevlahnota** [comment]
Yes they are automatically included as long as your pull request is active and you have changes in your fork.

**Comment by zethfoxster** [comment]
ok great, cause im going through the travis now fixing whatever its crying about. mostly unused variables and such.

**Pull request merged**